tests/Unit/Controller/PageControllerTest.php

```diff
@@ -189,7 +189,7 @@ public function testGenerateAuthorizationCode() {
 		// Wrong types
 		$result = $this->controller->generateAuthorizationCode(1, 'qwertz', 'abcd', 'state');
 		$this->assertInstanceOf(RedirectResponse::class, $result);
-		$this->assertEquals(OC_Util::getDefaultPageUrl(), $result->getRedirectURL());
+		$this->assertEqualsWithDelta(OC_Util::getDefaultPageUrl(), $result->getRedirectURL());
```

Comment:
```suggestion
		$this->assertEquals(OC_Util::getDefaultPageUrl(), $result->getRedirectURL());
```
IMO this line can be reverted.